dev/io.openliberty.pages.4.0.internal_fat/fat/src/io/openliberty/pages40/fat/tests/PagesELTests.java

```diff
@@ -0,0 +1,122 @@
+ * Copyright (c) 2013, 2023 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package io.openliberty.pages40.fat.tests;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.concurrent.Callable;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.logging.Logger;
+
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.api.spec.WebArchive;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+
+import com.ibm.websphere.simplicity.ShrinkHelper;
+import io.openliberty.pages40.fat.JSPUtils;
+import com.meterware.httpunit.GetMethodWebRequest;
+import com.meterware.httpunit.WebConversation;
+import com.meterware.httpunit.WebRequest;
+import com.meterware.httpunit.WebResponse;
+
+import componenttest.annotation.AllowedFFDC;
+import componenttest.annotation.ExpectedFFDC;
+import componenttest.annotation.Server;
+import componenttest.annotation.SkipForRepeat;
+import componenttest.custom.junit.runner.FATRunner;
+import componenttest.custom.junit.runner.Mode;
+import componenttest.custom.junit.runner.Mode.TestMode;
+import componenttest.rules.repeater.JakartaEEAction;
+import componenttest.topology.impl.LibertyServer;
+
+/**
+ *
+ */
+@RunWith(FATRunner.class)
+public class PagesELTests {
+    private static final Logger LOG = Logger.getLogger(PagesELTests.class.getName());
+    private static final String TestEL_APP_NAME = "TestEL";
+
+    @Server("pagesEL40Server")
+    public static LibertyServer server;
+
+    @BeforeClass
+    public static void setup() throws Exception {
+        ShrinkHelper.defaultDropinApp(server,
+                                      TestEL_APP_NAME + ".war",
+                                      "com.ibm.ws.jsp23.fat.testel.servlets");
+
+        server.startServer(PagesELTests.class.getSimpleName() + ".log");
+    }
+
+    @AfterClass
+    public static void tearDown() throws Exception {
+        if (server != null && server.isStarted()) {
+            server.stopServer();
+        }
+    }
+
+    /**
+     * Test JSP 2.3 Resolution of Variables and their Properties
+     *
+     * @throws Exception
+     *                       if something goes wrong
+     */
+    @Test
```

Comment:
This test exists elsewhere I think you skipped it and I asked why. So...I think I need some  clarification of what's going on with the tests and help you determine copyrights etc

Reply:
I updated this class with commented to state it was copied